routers/api/v1/repo/issue.go

```diff
@@ -336,9 +360,9 @@ func EditIssue(ctx *context.APIContext, form api.EditIssueOption) {
 			oneAssignee = *form.Assignee
 		}
 
-		err = models.UpdateAPIAssignee(issue, oneAssignee, form.Assignees, ctx.User)
+		err = issue_service.UpdateAssignees(issue, oneAssignee, form.Assignees, ctx.User)
```

Comment:
Did I miss the `models.CanBeAssigned()` for this call?

Reply:
For newly added assignees it is checked in UpdateAssignees->AddAssigneeIfNotAssigned. As this is not a new issue, there is no need to check before doing any transactions.